lib/autoInject.js

```diff
@@ -4,10 +4,10 @@ import arrayMap from 'lodash/_arrayMap';
 import clone from 'lodash/_copyArray';
 import isArray from 'lodash/isArray';
 
-var argsRegex =  /^function\s*[^\(]*\(\s*([^\)]*)\)/m;
+var argsRegex = /^(function[^\(]*)?\(?\s*([^\)=]*)/m;
 
 function parseParams(func) {
-    return  func.toString().match(argsRegex)[1].split(/\s*\,\s*/);
+    return func.toString().match(argsRegex)[2].trim().split(/\s*\,\s*/);
```

Comment:
That's one opinion.  But if you wish to see it removed, I think you should raise an issue ticket.  This is not the correct forum.